components/helpers/travelAdvice.ts

```diff
@@ -0,0 +1,21 @@
+import {DangerLevel} from 'types/nationalAvalancheCenter';
+
+export const TravelAdvice = (dangerLevel: DangerLevel): string => {
+  switch (dangerLevel) {
+    case DangerLevel.GeneralInformation:
+    case DangerLevel.None:
+      return 'Insufficient data for issuing of danger ratings, but a summary of avalanche conditions exists. Read the summary for more information.';
+    case DangerLevel.Low:
+      return '<strong>Generally safe avalanche conditions.</strong> Watch for unstable snow on isolated terrain features.';
+    case DangerLevel.Moderate:
+      return '<strong>Heightened avalanche conditions on specific terrain features.</strong> Evaluate snow and terrain carefully; identify features of concern.';
+    case DangerLevel.Considerable:
+      return '<strong>Dangerous avalanche conditions.</strong> Careful snowpack evaluation, cautious route-finding and conservative decision-making essential.';
```

Comment:
does `<strong>` work in React Native? I would have thought we'd need to return something like 
```suggestion
      return '<Caption1Semibold>Dangerous avalanche conditions.</Caption1Semibold> Careful snowpack evaluation, cautious route-finding and conservative decision-making essential.';
```

(but if it just works, then great)

Reply:
It does seem to work, but I am actually surprised - potentially we're wrapping this in an HTML renderer - should we just do full-fat text components here?